docker/server/Dockerfile

```diff
@@ -12,6 +12,8 @@ COPY . /conductor
 WORKDIR /conductor
 
 # Build the server on run
+RUN find . -name dependencies.lock -exec rm -f {} \;
```

Comment:
Just an advice:
it's better to do
```
RUN find . -name dependencies.lock -exec rm -f {} \ && \
         ./gradlew generateLock updateLock saveLock
         ./gradlew build -x test
```
in general such approach decreases final image size.

Reply:
> Just an advice:
> it's better to do
> 
> ```
> RUN find . -name dependencies.lock -exec rm -f {} \ && \
>     ./gradlew generateLock updateLock saveLock && \
>     ./gradlew build -x test
> ```
> 
> in general such approach decreases final image size.

[thanks](https://github.com/mendersoftware/conductor/pull/7/commits/1d7e8a87cd1c77877c67c7120f10ed35299aead4) for the advice!